microshift_release_notes/microshift-4-16-release-notes.adoc

```diff
@@ -77,8 +77,12 @@ The following list provides update details:
 With this release, you can use the GitOps with Argo CD agent derived from GitOps 1.12 with {microshift-short}. Using GitOps means you can update a single Git repository and automate the deployment of new applications or updates to existing ones. You can also use your Git repository as an audit trail of changes so that you can create processes such as review and approval for merging pull requests that implement configuration changes.
 //TODO add xrefs once docs merge
 
-//[id="microshift-4-16-support"]
-//=== Support
+[id="microshift-4-16-support"]
+=== Support
+
+[id="microshift-4-16-support-updates"]
+==== Getting cluster ID
+With this release, you can get cluster ID for the Microshift clusters using kube-system namespace. See Getting your cluster ID for more information.
```

Comment:
```suggestion
With this release, you can get the cluster ID for a {microshift-short} instance using `kube-system` namespace. When opening a support case, it is helpful to provide debugging information to Red Hat Support, including the unique identifier for your cluster.
```
Aside from the support case, is the cluster ID helpful to the customer in any immediate way? And I would leave the "See the section" out until we have the xref.

Reply:
Cluster ID is used only in support cases. So I have made the suggested changes.